datastore/longhorn.go

```diff
@@ -1014,23 +1014,23 @@ func tagNodeLabel(nodeID string, obj runtime.Object) error {
 func (s *DataStore) GetSettingAsInt(settingName types.SettingName) (int64, error) {
 	definition, ok := types.SettingDefinitions[settingName]
 	if !ok {
-		return 0, fmt.Errorf("setting %v is not supported", settingName)
+		return -1, fmt.Errorf("setting %v is not supported", settingName)
```

Comment:
You don't need to change this. The error has been returned so the returned value should be ignored.

Reply:
In fact, this one is fine. It's better to set to an invalid number. Just don't use it in place of an error.